findingcheapestgpus.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: 'Finding the Cheapest GPUs'
+---
+Shadeform makes it easy to find and deploy VM and containers to the cheapest GPU instances across many different clouds. Let's show how we can do this with the UI and the API, specifically for an instance with a single 80 GB A100 GPU.
+
+
+## Using the Shadeform UI
+
+### 1. Searching for GPU's
+Once you've created an account and added funds to your [Wallet](https://platform.shadeform.ai/settings/billing) then you can see available instances on the homepage. 
```

Comment:
You can see available instances on the homepage without topping the wallet. Can we move the topping up the wallet to the "Launching a GPU" section?

Reply:
pinging @esgoode here. I got the impression that you all wanted the platform to be more about _using_ the GPU's that are found via the platform (requiring a wallet) than just using Shadeform as a pricing page and then going direct to underlying providers. So I included the part about the wallet. Happy to change, lmk